### DISCUSSION TITLE
Combining RNA and ATAC

### DISCUSSION THREAD

**Discussion by francescasci91** [satijalab/seurat#6062] [Q&A] [answered]
Hi, I'm trying to make a joint analysis of RNA and ATAC data but I'm having troubles.
I have 6 datasets of RNA and 4 of ATAC.
This is what I tried to do
1) Integration of RNA datasets using SCT transformation v2
2) Integration of ATAC following the standard pipeline vignette

The problem comes when I want to merge the two modalities  to go through WNN analysis.

I tried merging the two integrated datasets with the merge function and it appears a warning 
`“Attempting to merge an SCTAssay with another Assay type Converting all to standard Assay objects.”
`
So, when I call `FindMultiModalNeighbors`, this error comes out :

`Error: Cannot find 'pca' in this Seurat object`

Basically, I think that the PCA slot of SCT assay is removed. 

What can I do?

Thanks
Francesca

**Comment by timoast** [comment, marked as answer]
Answered here https://github.com/timoast/signac/discussions/1140
